backend/src/routes/depositRoute.js

```diff
@@ -41,5 +41,13 @@ depositRoute.get('/:userId', async (req, res) => {
     res.status(500).send('Something went wrong, please try again later.');
   }
 });
+depositRoute.get('/:userId/:depositId', async(req, res)=>{
```

Comment:
```suggestion
depositRoute.get('/:depositId', async(req, res)=>{
```
This is no longer the way the service works, user id comes from the token